xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -92,11 +92,17 @@ void CGUIWindowPVRGuide::GetContextButtons(int itemNumber, CContextButtons &butt
     CPVRTimerInfoTagPtr timer(epg->Timer());
     if (timer)
     {
+      if (timer->GetTimerScheduleId() != PVR_TIMER_NO_PARENT)
+        buttons.Add(CONTEXT_BUTTON_EDIT_TIMER_RULE, 19243); /* Edit timer rule */
```

Comment:
Then, I would like to leave this as it is now and to do it the other way round in another PR: change internal naming  from "repeating timer" to "timer rule", for the sake of consistency between UI and internal naming. Okay?